### DISCUSSION TITLE
Category home deleted by mistake

### DISCUSSION THREAD

**Discussion by paulandriantefy** [PrestaShop/PrestaShop#36814] [Q&A]
Hello,
I deleted home category by mistake, the problem is that I don't have access to the database to add the home category.
Is there an issue to restore it without database access please? thank you

**Comment by kpodemski** [comment]
Do you have access to PrestaShop files? If you do, you can interact with the database directly from the files, or even decide to install phpMyAdmin or Adminer script.

**Reply by paulandriantefy** [reply]
No, I don't
